package.json

```diff
@@ -183,14 +184,12 @@
     "postinstall": "node ./scripts/postinstall.mjs",
     "lint": "eslint --format ./scripts/eslint-formatter.cjs",
     "stylelint": "stylelint --quiet --allow-empty-input \"packages/**/*.css\" \"docs/**/*.css\"",
-    "test": "node --max_old_space_size=4096 node_modules/@ckeditor/ckeditor5-dev-tests/bin/testautomated.js",
-    "manual": "node --max_old_space_size=8192 node_modules/@ckeditor/ckeditor5-dev-tests/bin/testmanual.js",
+    "test": "yarn ckeditor5-dev-tests-run-automated",
+    "manual": "yarn ckeditor5-dev-tests-run-manual",
     "manual:verify": "node ./scripts/web-crawler/index.mjs --manual",
-    "bootstrap": "yarn install",
     "clean": "npx rimraf --glob package-lock.json yarn.lock ./**/node_modules",
-    "reset": "rm -rf ./node_modules && yarn run bootstrap",
-    "reinstall": "yarn run clean && yarn run bootstrap",
-    "docs": "node --max-old-space-size=8192 scripts/docs/build-docs.mjs",
```

Comment:
Is it safe to remove `--max-old-space-size=8192`? Wouldn't it cause the docs build to be unstable?

Reply:
It works fine for me and on CI. I assume it was necessary when we were building snippets with webpack.